src/fastga_he/powertrain_builder/unit_tests/test_power_train_file.py

```diff
@@ -166,6 +175,157 @@ def test_power_train_file_connections():
         print("[" + om_output + ", " + om_input + "]")
 
 
+def test_power_train_file_cache():
```

Comment:
This checks that redoing the _get_connections() does not add a new item to the cache if it has already been checked right ? 
Also I'm now thinking that `_cache` is not very explicit maybe `_connection_check_cache`